transcom/mymove
# B-17953 Update status after converting to customer expense

## Summary

- Created a PATCH request for updating the `convertToCustomerExpense` and `customerExpenseReason` values when the TOO chooses to convert current SIT to customer expense. These values are within `MTOServiceItems` model.
- `Converted to customer expense` Tag has also been added to show the status text in the SIT module.

### How to test

1. Create a test move by using `HHGMoveInSITEndsTomorrow` button in the `Testharness Data Scenarios` page.
2. As a TOO, go to the move and on "Move task order" tab, click the link "Convert to customer expense" and fill out the form then save.
3. You should see the `CONVERTED TO CUSTOMER EXPENSE` tag in yellow.

## Screenshots
<img width="768" alt="image" src="https://github.com/transcom/mymove/assets/146856854/bc2fc277-4ba5-4c1d-a9fa-107e030a6e6f">
